analysis/study_definition.py

```diff
@@ -473,6 +476,25 @@
             },
         ),
     ),
+    # High blood pressure or hypertension
+    bp_ht=patients.categorised_as(
+        {
+            "0": "DEFAULT",
+            "1": """
+                    (bp_sys >= 140 OR bp_dia >= 90) OR
+                        hypertension
+            """,
+        },
```

Comment:
This should work, but it would be marginally neater to use [`patients.satisfying`](https://docs.opensafely.org/study-def-variables/#cohortextractor.patients.satisfying), which is really just a special case of `patients.categorised_as` that returns a binary variable. No need to change now if you'd like to get it run.

Reply:
I think this will return the same:
```py
bp_ht=patients.satisfying(
        "bp_sys >= 140 OR bp_dia >= 90 OR hypertension",
```

Reply:
Thx - didn't know - i've implemented this now